scripts/linters/pre_commit_linter.py

```diff
@@ -576,6 +591,8 @@ def main(args=None):
     """Main method for pre commit linter script that lints Python, JavaScript,
     HTML, and CSS files.
     """
+    name_space = multiprocessing.Manager().Namespace()
```

Comment:
Could you add a comment explaining the usage of the namespace in more detail? I see it scattered throuhgout but I don't really understand what we're doing with it. Also why do we need to pass it into everything -- can it be something more "global" instead?

Reply:
Added an explanation.

Previously it was a global, the problem is that in Python 3 when global variable was used the process handged and test couldn't finish. Rn, I can only find this issue for a reference https://bugs.python.org/issue15914 (it is mentioning `SyncManager` but `Namespace` uses `SyncManager`).